cherab/openadas/rates/pec.pyx

```diff
@@ -20,10 +20,13 @@ import numpy as np
 
 from libc.math cimport INFINITY, log10
 
-from raysect.core.math.function.float cimport Interpolator2DArray
+from raysect.core.math.function.float cimport Interpolator2DArray, Interpolator3DArray
 from cherab.core.utility.conversion import PhotonToJ
 
 
+DEF ZERO_THRESHOLD = 1.e-300
```

Comment:
Does this need to be >0?

Also, prefer `cdef ZERO_THRESHOLD` to `DEF`: https://cython.readthedocs.io/en/latest/src/userguide/migrating_to_cy30.html#deprecation-of-def-if. Or just type the value out manually: it's only used in 3 places within the same function.

Reply:
Initially I introduced this threshold when we switched to Raysect interpolators and moved to log-log interpolation for atomic rates. The idea was to not override the extrapolation settings of the interpolators by forcibly returning zero when extrapolation is disabled. However, if we decided to return zero rate for zero plasma parameters, it is safe to remove the threshold and just check if parameters are <= 0.